tests/test_validation/steps/test_metadata_reader.py

```diff
@@ -129,11 +129,10 @@ def test_read_invalid_unit_type():
     with pytest.raises(ValidationError) as e:
         metadata_reader.run_reader(DATASET_NAME, METADATA_PATH)
     assert "Errors found while validating metadata file" in str(e)
-    assert e.value.errors == [
+    assert (
         "identifierVariables->unitType: value is not a valid enumeration member; "
         "permitted: 'JOBB', 'KJORETOY', 'FAMILIE', 'FORETAK', 'HUSHOLDNING', "
```

Comment:
Are the rest of permitted values missing?
"'KOMMUNE', 'KURS', 'PERSON', 'VIRKSOMHET', 'BK_HELSESTASJONSKONSULTASJON'"

Reply:
I rewrote it so it doesn't include all the permitted values but ensures you get the correct error message. Or else we have to update this unit test message check every time we add a new unit_type 👍🏻